CONTRIBUTING.rst

```diff
@@ -146,7 +145,7 @@ To set up ScanCode for local development:
 
         or
 
-        . venv/bin/activate
+        .venv/bin/activate
```

Comment:
This is wrong. Try it out and you will see

Reply:
Fixed.